packages/codd/hooks/codd-validate-precommit.py

```
@@ -60,6 +225,147 @@
+# `--pathspec-from-file`。B-2: Issue #338 反復4 bot レビュー対応）。
+_COMMIT_UNSUPPORTED_LONG_FLAGS = {
+    "--patch",
+    "--interactive",
+    "--include",
+    "--only",
+    "--pathspec-from-file",
+}
+# 再現困難な long option のうち、値を取るもの（"=value" 形式でなければ次トークンを
+# 値として読み飛ばす。B-2）。
+_COMMIT_UNSUPPORTED_VALUE_LONG_FLAGS = {"--pathspec-from-file"}
+# 値を取る短縮オプション文字（結合形の途中に現れたら、以降を attached value として
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)